abcd/backends/atoms_opensearch.py

```diff
@@ -385,7 +386,7 @@ def push(
         Parameters
         ----------
         atoms: Union[Atoms, Iterable]
-        extra_info: Union[dict, str, None], optional
+        extra_info: Optional[dict, str], optional
```

Comment:
Not sure this is right (not sure it's necessarily wrong), but should it be:
```suggestion
        extra_info: Optional[Union[dict, str]], optional
```
Since `Optional` is shorthand for `Union[..., None]`, I actually think the original typehints are fine if you don't want to be extra specific.

Reply:
Ah yes, got carried away with replacing, thanks